impl/pom.xml

```diff
@@ -27,17 +27,23 @@
    <modelVersion>4.0.0</modelVersion>
 
    <parent>
-        <groupId>org.glassfish</groupId>
+        <groupId>org.wildfly.faces</groupId>
```

Comment:
I think we can stick to `com.sun.faces`.

Reply:
Fixed